src/libsync/theme.cpp

```diff
@@ -412,5 +412,10 @@ QString Theme::webDavPath() const
     return QLatin1String("remote.php/webdav/");
 }
 
+QString Theme::webDavPathNonShib() const
+{
+    return QLatin1String("remote.php/nonshib-webdav/");
```

Comment:
This is a custom endpoint you patched into the server?
Or is that also what @DeepDiver1975 is using?